src/main/java/org/example/discstorage/UserInfoDiscStorageHandler.java

```diff
@@ -0,0 +1,49 @@
+package org.example.discstorage;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.Set;
+import java.util.TreeSet;
+import org.example.exceptions.InvalidArgument;
+import org.example.model.User;
+
+public class UserInfoDiscStorageHandler {
```

Comment:
storage should always have interface, currently you're using file storage but we may change it to sql in future. So storageHandler should have two functionality that is save and get and fileStorageHandler should implement that

Reply:
created interface for this